### ISSUE TITLE
Adding galleries does not work

### ISSUE THREAD

**Issue by n0k0ut** [Pewpews/happypanda#73]
Whenever I attempt to add an archive or gallery using any of the methods available, the program shows the "nothing was added" error message. I'm able to navigate to the directory using the file explorer, and changing permissions does not seem to fix the problem.

[happypanda_debug.log.txt](https://github.com/Pewpews/happypanda/files/80652/happypanda_debug.log.txt)

**Comment by twiddli** [comment]
Hello and thank you for reporting! 
Are you aware of the option "Subdirectories should be treated as standalone galleries..." in settings? I'd just like to make sure. 

When you try to select an archive or folder in the "Add single gallery" dialog, does it show any red text? If so, what does it say?

**Comment by ghost** [comment]
In the log, this is the line 376:

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/me/documents/japanese/reading material/manga/yuru yuri/ゆるゆり 第02巻.cbz'
```

And this is the line 377:

```
06-01 20:14 WARNING fetch  Skipped b'/home/me/documents/japanese/reading material/manga/Yuru Yuri/\xe3\x82\x86\xe3\x82\x8b\xe3\x82\x86\xe3\x82\x8a \xe7\xac\xac02\xe5\xb7\xbb.cbz' in local search
```

Notice how the first line says `yuru yuri`, while the second says `Yuru Yuri`.

Some systems handle file names in a case-sensitive way, so IMHO it's possible that [this line in `version/utils.py` (`check_archive`)](https://github.com/Pewpews/happypanda/blob/93f2c916f2fb464aaddb192b6f0318cc4e94c804/version/utils.py#L429) might be causing this.

The problem happens [when trying to instance a ZipFile](https://github.com/Pewpews/happypanda/blob/93f2c916f2fb464aaddb192b6f0318cc4e94c804/version/utils.py#L290): it can't find the file. And [`os.path.normcase`](https://docs.python.org/3/library/os.path.html#os.path.normcase) is returning the path unchanged, because OP is using a case-sensitive system.

Basically, unless you're using a case-insensitive system, any path that is not entirely lowercase will be invalid.

I think the fix would be changing this in `version/utils.py`:

```
429c429
<       archive_path = archive_path.lower()
---
>       archive_path = os.path.normcase(archive_path)
```

**Comment by twiddli** [comment]
Nicely spotted!
Thanks for the pointers. I've changed the line.
Let's just hope that was the real cause!

EDIT: 
Removed the line. It was actually unnecessary in the first place.
